web_src/js/features/pull-view-file.js

```diff
@@ -43,9 +43,11 @@ export function initViewedCheckboxListenerFor() {
       // Mark the file as viewed visually - will especially change the background
       if (this.checked) {
         form.classList.add(viewedStyleClass);
+        checkbox.setAttribute('checked', '');
```

Comment:
```suggestion
        checkbox.checked = true;
```

Reply:
Actually comments mention `change` does not trigger with `.checked`, so likely your method is better.